### PR TITLE
XSD import fix

### PR DESCRIPTION
**Pull request by SimonCockx**
Please include a summary of the change and the issue/story number.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

### CODE REVIEW

File: rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBuilderGenerator.xtend [hunk 1]
```diff
@@ -173,6 +174,7 @@ class ModelObjectBuilderGenerator {
 		'''
 		«IF attribute.isMulti»
 			@Override
+			@«RosettaAttribute»("«attribute.javaAnnotation»")
```
Comment (SimonCockx) [inline rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBuilderGenerator.xtend:177]: Moved `@RosettaAttribute` from `setListAttribute` to `addListAttribute`, which is the convention in Jackson.

Comment (SimonCockx) [reply rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBuilderGenerator.xtend:177]: Alternatively, I could modify the default Jackson behaviour in rosetta-common?

Comment (SimonCockx) [reply rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/ModelObjectBuilderGenerator.xtend:177]: See diffs in `PojoRegressionTest`

**Review by hugohills-regnosys — APPROVED** [review]

**Comment by hugohills-regnosys** [comment]
https://github.com/finos/common-domain-model/issues/3364

**Pull request merged**